fastly/blobstorage.go

```diff
@@ -27,6 +27,7 @@ type BlobStorage struct {
 	MessageType       string     `mapstructure:"message_type"`
 	Placement         string     `mapstructure:"placement"`
 	ResponseCondition string     `mapstructure:"response_condition"`
+	FileMaxBytes      uint       `mapstructure:"file_max_bytes"`
```

Comment:
NOTE: this setting is currently not documented: 
https://developer.fastly.com/reference/api/logging/azureblob/ 

But there is an internal ticket for adding it to the API docs and that ticket is currently 'in review'.

Reply:
The docs PR is approved and I expect it'll get deployed very soon.